apilint/src/test/resources/apilint_test/test-annotation-arguments-androidx.txt

```diff
@@ -0,0 +1,15 @@
+package test {
+  public abstract static class Test.TestClass<Settings extends java.lang.String> {
+    method @android.support.annotation.MainThread public final void testMethod(@android.support.annotation.NonNull java.lang.String, @android.support.annotation.NonNull java.lang.String);
+    method @android.support.annotation.MainThread @android.support.annotation.NonNull public Settings testGeneric();
+    method @android.support.annotation.MainThread @android.support.annotation.NonNull public java.lang.Map<java.lang.Map<Settings, Settings>, java.lang.Map<Settings, Settings>> testMultipleGeneric();
+  }
+
+  public abstract static class RuntimeSettings.Builder<Settings extends org.mozilla.geckoview.RuntimeSettings> {
+    ctor public Builder();
+    method @android.support.annotation.AnyThread @android.support.annotation.NonNull public Settings build();
+    method @android.support.annotation.AnyThread @android.support.annotation.NonNull protected Settings getSettings();
+    method @android.support.annotation.AnyThread @android.support.annotation.NonNull protected abstract Settings newSettings(@android.support.annotation.Nullable Settings);
```

Comment:
shouldn't this use `@androidx.annotation.*` instead of `@android.support.annotation.*`